charts/newrelic-infrastructure/templates/controlplane/_agent-config_helper.tpl

```diff
@@ -0,0 +1,20 @@
+{{- /*
+Defaults for controlPlane's agent config
+*/ -}}
+{{- define "nriKubernetes.controlPlane.agentConfig.defaults" -}}
+is_forward_only: true
+is_secure_forward_only: true
```

Comment:
It worked in the end-to-end because the agent ends adding plugins when the `is_forward_only` is set to true and it does not care about if it is secure or not. [See this snippet.](https://github.com/newrelic/infrastructure-agent/blob/master/pkg/plugins/plugins_linux.go#L42-L59)

So if `is_forward_only` is set to true there is no need to ser `is_secure_forward`.

I have to say that I added using [the `Dockerfile` from the forwarder as an example](https://github.com/newrelic/infrastructure-agent/blob/master/build/container/Dockerfile#L47-L66) to set these environment variables.

As I said in other thread:
> Is also a step to remove the forwarder image and use agent one directly if we want to remove spurious hard-to-follow dependencies.

I can clean this a bit. Fixed in [#1cfa609](https://github.com/newrelic/nri-kubernetes/pull/383/commits/1cfa609a444fa9ffffec3c83b67469243ef5ab56)

I am leaving the thread open in case you want to blame something more :)